content/pt/docs/concepts/cluster-administration/addons.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Instalando Addons
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+
+Addons estendem a funcionalidade do Kubernetes.
+
+Esta página lista alguns dos add-ons and links com suas respectivas instruções de instalação.
+
+Add-ons em cada seção são classificados em ordem alfabética - a encomenda não implica qualquer status preferencial.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Rede e Política de Rede
+
+
+* [ACI](https://www.github.com/noironetworks/aci-containers) fornece rede integrada de contêineres e segurança de rede com a Cisco ACI.
+* [Calico](https://docs.projectcalico.org/latest/getting-started/kubernetes/) é um provedor de políticas de rede e rede L3 seguro.
+* [Canal](https://github.com/tigera/canal/tree/master/k8s-install) une Flannel e Calico, fornecendo rede e política de rede.
+* [Cilium](https://github.com/cilium/cilium) é um plug-in de políticas de rede e rede L3 que pode impor políticas de HTTP / API / L7 de forma transparente. Tanto o modo de roteamento quanto o de sobreposição / encapsulamento são suportados.
+* [CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie) permite que o Kubernetes se conecte facilmente a uma variedade de plugins CNI, como Calico, Canal, Flannel, Romana ou Weave.
+* [Contiv](http://contiv.github.io) fornece um rede configurável (L3 nativa usando BGP, sobreposição usando vxlan, L2 clássico e Cisco-SDN / ACI) para vários casos de uso e uma estrutura rica de políticas de rede. O projeto Contiv é totalmente [open source](http://github.com/contiv). O script de[instalação](http://github.com/contiv/install) fornece opções de instalação com ou sem kubeadm.
+* [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), baseado no [Tungsten Fabric](https://tungsten.io), é um projeto open source, multi-cloud com uma rede virtualizada e com uma plataforma de gerenciamento de políticas de rede. Contrail and Tungsten Fabric are integrated with orchestration systems such as Kubernetes, OpenShift, OpenStack and Mesos, and provide isolation modes for virtual machines, containers/pods and bare metal workloads.
```

Comment:
I believe has more text to Portuguese translation on the description.

Reply:
good catch, thanks